### DISCUSSION TITLE
Release Status Provider?

### DISCUSSION THREAD

**Discussion by vfweiss** [SAP/terraform-provider-cloudfoundry#190] [Q&A] [answered]
Dear SAP,

What is the current status of the provider? Would you say it's production ready already? If not, is there a schedule for the provider when it will be? How stable is the current setup?

We are starting to run into issues with the Community Provider and it's usage of the V2 API and the rate limit applied for it.  So if possible we would rather sooner than later switch to this provider. However are not looking forward to do a migration more than once. With the BTP provider we ran into the breaking changes and that was quite a nuisance.

Kind regards,

Vincent Weiss

**Comment by lechnerc77** [comment, marked as answer]
Hi @vfweiss 

thanks for the message: It was a bit quiet due to the fact that we donated the provider to the place where it belongs to, namely the Cloud Foundry Foundation.

- Link to Repository: <https://github.com/cloudfoundry/terraform-provider-cloudfoundry>
- Link to Terraform Registry <https://registry.terraform.io/providers/cloudfoundry/cloudfoundry/latest>

Please use this provider as of now. License-wise the move does not cause any changes. Same is true for the support channels (i.e., GitHub issues). 

The provider is available as RC1, so you should give it a try and validate that all obstacles you are facing with the other one are resolved. If not please open an issue in the repository <https://github.com/cloudfoundry/terraform-provider-cloudfoundry/issues>.
We are in the final touches to provide a GA release of the provider, so it is a matter of a few weeks that a GA version is available. 
 
We will update the `README` of this repository in the `SAP` organization on GitHub soon to point you to the right spots with regards to the Terraform registry as well as the source code.
